templates/repository/common/CONTRIBUTING.md

```diff
@@ -55,7 +55,7 @@ do this is via the [ORY Community](https://community.ory.sh/) or join the
 ## FAQ
 
 - I am new to the community. Where can I find the
-  [ORY Community Code of Conduct?](CODE_OF_CONDUCT.md)
+  [ORY Community Code of Conduct?](https://github.com/ory/meta/blob/master/CODE_OF_CONDUCT.md)
```

Comment:
We should link to the project's code of conduct.

Reply:
Then we have to copy the code of conduct into each projects /docs too right?

Reply:
https://github.com/ory/hydra/blob/master/CODE_OF_CONDUCT.md